<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>IN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="port_box_in">
        <item>
         <property name="text">
          <string>COM1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM12</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Baudrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="boud_in">
        <property name="currentText">
         <string>1200</string>
        </property>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connect_button_in">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>100</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnect_button_in">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>100</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>OUT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="port_box_out">
        <item>
         <property name="text">
          <string>COM1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM12</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Baudrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="boud_out">
        <property name="currentText">
         <string>1200</string>
        </property>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connect_button_out">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>100</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnect_button_out">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>100</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="port_box_send">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>COM1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM12</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="box_priority">
        <property name="text">
         <string>set priority</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="send_marker">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Send marker</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="priority">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_far_station">
        <property name="text">
         <string>Far station</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring></cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="far_station">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="if_bit_staffing">
        <property name="text">
         <string>BitStuffing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="staffing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>01111110</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="if_crc">
        <property name="text">
         <string>CRC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="polinom">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>100011011</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="if_power">
        <property name="text">
         <string>power</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="chat"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mesage"/>
      </item>
      <item>
       <widget class="QPushButton" name="send_button">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
